website/docs/guides/container_service_extension_4_0.html.markdown

```diff
@@ -0,0 +1,411 @@
+
+### Step 1: Create RDEs and the CSE Administrator user
+
+-> This step of the installation refers to the Terraform configuration present [here][step1].
+
+In the [given configuration][step1] you can find a file named `terraform.tfvars.example`, you need to rename it to `terraform.tfvars`
+and change the values present there to the ones that fit with your needs.
+
+This step will create the following:
+
+- The required `VCDKEConfig` [RDE Interface][rde_interface] and [RDE Type][rde_type]. These two resources specify the schema of the CSE Server
+  configuration (called "VCDKEConfig") that will be instantiated in next step with a [RDE][rde].
+- The required `capvcdCluster` [RDE Type][rde_type]. Its version is specified by the `capvcd_rde_version` variable, that **must be "1.1.0" for CSE v4.0**.
+  This resource specifies the schema of the TKGm clusters.
+- The **CSE Admin [Role][role]**, that specifies the required rights for the CSE Administrator to manage provider-sided elements of VCD.
+- The **CSE Administrator [User][user]** that will administrate the CSE Server and other aspects of VCD that are directly related to CSE.
+  Feel free to add more attributes like `description` or `full_name` if needed.
+
+Once reviewed and applied with `terraform apply`, one **must login with the created CSE Administrator user to
+generate an API token** that will be used in the next step. In UI, the API tokens can be generated in the CSE Administrator
+user preferences in the top right, then go to the API tokens section, add a new one.
+Or you can visit `/provider/administration/settings/user-preferences` from your VCD url as CSE Administrator.
+
+### Step 2: Install CSE
+
+-> This step of the installation refers to the Terraform configuration present [here][step2].
+
+~> Be sure that previous step is successfully completed and the API token for the CSE Administrator user was created.
+
+This step will create all the remaining elements to install CSE v4.0 in VCD. You can read subsequent sections
+to have a better understanding of the building blocks that are described in the [proposed Terraform configuration][step2].
+
+In this [configuration][step2] you can also find a file named `terraform.tfvars.example`, you need to rename it to `terraform.tfvars`
+and change the values present there to the correct ones. You can also modify the proposed resources so they fit better to your needs.
+
+#### Organizations
+
+The [proposed configuration][step2] will create two new [Organizations][org], as specified in the [CSE documentation][cse_docs]:
+
+- A Solutions [Organization][org]: This [Organization][org] will host all provider-scoped items, such as the CSE Server.
+  It should only be accessible to the CSE Administrator and Providers.
+- A Tenant [Organization][org]: This [Organization][org] will host the TKGm clusters for the users of this tenant to consume them.
```

Comment:
I usually like links a lot, but in this case it's a bit too much for the same link :) Suggesting to simplify for the paragraph to feel more "soft":

```suggestion
- A Solutions [Organization][org], which will host all provider-scoped items, such as the CSE Server.
  It should only be accessible to the CSE Administrator and Providers.
- A Tenant [Organization][org], which will host the TKGm clusters for the users of this tenant to consume them.
```

p.s. "TKGm clusters", on the other hand, could have a link, if you have one handy.

Reply:
Done! Added the TKGm links also.